docs/index.rst

```diff
@@ -5,66 +5,85 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
```

Comment:
I really like this organization of the index page. Everything is more searchable and structured.

Reply:
Thanks. :-)

Reply:
I agree!